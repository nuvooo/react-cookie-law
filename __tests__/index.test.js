import React from 'react';
import expect from 'expect';
import CookieBanner from '../index';

describe('CookieBanner', () => {
  it('renders correctly', () => {
    const wrapper = <CookieBanner cookieHeadline="Test" cookieMessage="Hello World!" />;
    expect(wrapper).toMatchSnapshot();
  });
});
