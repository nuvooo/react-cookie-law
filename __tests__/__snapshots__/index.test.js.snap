// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CookieBanner renders correctly 1`] = `
<CookieLaw
  cookieHeadline="Test"
  cookieMessage="Hello World!"
/>
`;
